kpops/cli/pipeline_config.py

```diff
@@ -20,19 +24,20 @@ class TopicNameConfig(BaseSettings):
     )
 
 
-class KafkaConnectResetterConfig(BaseSettings):
-    helm_config: HelmConfig = Field(
-        default=HelmConfig(
+class KafkaConnectResetterHelmConfig(BaseSettings):
+    helm_config: HelmRepoConfig = Field(
+        default=HelmRepoConfig(
             repository_name="bakdata-kafka-connect-resetter",
             url="https://bakdata.github.io/kafka-connect-resetter/",
-            version="1.0.4",
         ),
         description="Configuration of Kafka connect resetter Helm Chart",
     )
+    version = "1.0.4"
```

Comment:
this requires a type annotation because dataclasses / Pydantic models can't just infer them